src/App.tsx

```diff
@@ -12,6 +13,21 @@ const Div = styled.div`
 
 export const App = () => {
   if (typeof window === "undefined") return null; // make sure latest + greatest browser info
+
+  // polyfill crypto.randomUUID
```

Comment:
didn't even consider a polyfill, I suppose browser support isn't 100% so nice addition https://caniuse.com/mdn-api_crypto_randomuuid

Reply:
Yeah I wasn't even aware of crypto.randomUUID before this. Cool to learn about!